cmd/restic/cmd_fsck.go

```diff
@@ -153,11 +148,11 @@ func fsckTree(opts CmdFsck, s *server.Server, blob server.Blob) error {
 	}
 
 	// check map for unused ids
-	for _, id := range tree.Map.IDs() {
-		if seenIDs.Find(id) != nil {
-			return fmt.Errorf("tree %v: map contains unused ID %v", blob.ID, id)
-		}
-	}
+	// for _, id := range tree.Map.IDs() {
+	// 	if seenIDs.Find(id) != nil {
+	// 		return fmt.Errorf("tree %v: map contains unused ID %v", id, id)
+	// 	}
+	// }
```

Comment:
fsck is not yet fixed for the new repository structure with packs, I have to work on that.